dvl-a50/mavlink2resthelper.py

```diff
@@ -352,14 +352,36 @@ def send_vision_position_estimate(
         )
         logger.info(post(MAVLINK2REST_URL + "/mavlink", data=data))
 
-    def send_rangefinder(self, distance: float):
+    def send_rangefinder(self, distance: float, sensor_id: int = 0, orientation: str = "MAV_SENSOR_ROTATION_PITCH_270"):
         "Sends message DISTANCE_SENSOR to flight controller"
         if distance == -1:
             return
-        data = self.rangefinder_template.format(int(distance * 100))
-
+        data = self.rangefinder_template.format(
+            distance=int(distance * 100), 
+            sensor_id=sensor_id, 
+            orientation=orientation
+        )
         post(MAVLINK2REST_URL + "/mavlink", data=data)
 
+    def send_beam_distances(self, beam_distances: list, beam_valid: list):
+        """
+        Send individual beam distances from DVL transducers as separate DISTANCE_SENSOR messages
+        Each beam gets a different orientation to work around ArduSub's single rangefinder per orientation limitation
+        """
+        # Define orientations for each beam (1=rear-right, 2=rear-left, 3=front-left, 4=front-right)
+        # These orientations represent the 4 quadrants around the DVL
+        beam_orientations = [
+            "MAV_SENSOR_ROTATION_YAW_135",  # rear-right
+            "MAV_SENSOR_ROTATION_YAW_225",  # rear-left
+            "MAV_SENSOR_ROTATION_YAW_315",  # front-left 
+            "MAV_SENSOR_ROTATION_YAW_45"    # front-right
```

Comment:
This doesn't account for the DVL orientation, which seems problematic when other parts of the codebase allow for `DVL_DOWN` as well as `DVL_FORWARD` options.

That said, if I'm remembering correctly the forward orientation isn't currently exposed through the frontend interface, so if you don't want to implement support for it then I think it's fine to just put a `TODO` comment for what needs doing if/when DVL_FORWARD support eventually gets enabled.